files/brews/hub.rb

```diff
@@ -0,0 +1,22 @@
+require 'formula'
+
+class Hub < Formula
+  VERSION = "2.2.0-rc1"
```

Comment:
Is the constant needed here? Because Homebrew auto-detect the version from the URL, I think it also exposes that formula property at runtime so you might be able to use `version` during the `test` block